nix/overlay.nix

```diff
@@ -89,6 +89,12 @@ self: super: {
     ];
   };
 
+  haskellPackages = super.haskellPackages.override {
+    overrides = hself: hsuper: {
+      weeder = self.haskell.lib.dontCheck (hself.callPackage ./pkgs/weeder { });
+    };
```

Comment:
We should write the reason we're using a fork and link to a PR if we have expectation that this is temporary.